control-plane-operator/hostedclusterconfigoperator/cmd.go

```diff
@@ -208,6 +214,16 @@ func (o *HostedClusterConfigOperator) Run(ctx context.Context) error {
 			"konnectivity-agent": konnectivityAgentImage,
 		},
 	}
+	kubeClient, err := kubernetes.NewForConfig(cpConfig)
+	if err != nil {
+		setupLog.Error(err, "unable to create kube client")
+		os.Exit(1)
+	}
+	activeImage, err := util.LookupActiveContainerImage(context.TODO(), kubeClient.CoreV1().Pods(o.Namespace), os.Getenv("POD_NAME"), "hosted-cluster-config-operator")
```

Comment:
Should probably use a timeout context based on `ctx`

Reply:
Agreed changing

Reply:
for both control-plane-operator and hosted-cluster-config-operator

Reply:
done